recipes/opengv/all/conandata.yml

```
@@ -2,3 +2,8 @@ sources:
   "cci.20200806":
     url: "https://github.com/laurentkneip/opengv/archive/91f4b19c73450833a40e463ad3648aae80b3a7f3.tar.gz"
     sha256: "b03f61ff597a6c16a32b8939c3e49e9f240ae7b4da3358f1430e9743e093d596"
+patches:
+  "cci.20200806":
+    - patch_file: "patches/0001-use-conans-pybind11.patch"
+    - patch_file: "patches/0002-use-conans-eigen.patch"
+    - patch_file: "patches/0003-let-conan-handle-shared-and-fpic.patch"
```

https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patch_description

```suggestion
    - patch_file: "patches/0001-use-conans-pybind11.patch"
       patch_description: "fix call to pybind to use conan provided config file"
       patch_type: "conan"
    - patch_file: "patches/0002-use-conans-eigen.patch"
       patch_description: "fix call to find_package to use conan provided file"
       patch_type: "conan"
    - patch_file: "patches/0003-let-conan-handle-shared-and-fpic.patch"
       patch_description: "disable some options such that conan manages values"
       patch_type: "conan"
```